### PR TITLE
#22508 Add User into the Manifest

### LINKED ISSUE #22149: The push publish button in the expansion menu on the users portlet doesn't just push the selected user, it pushes all users
**Describe the bug**
When you utilize the menu in the top right corner of the users portlet to push publish, it pushes every user and not only the selected one.

image

Reproduced on: 22.03, 21.06
Ticket: https://dotcms.zendesk.com/agent/tickets/107128

**To Reproduce**
1. Set up a sender/receiver
2. Go to the users portlet
3. Select a user
4. Click the 'Push Publish' button in the top right menu, every user will be included in the bundle

**Expected behavior**
The button should either only push the selected user, or it should be explicit that it will push every user.

### LINKED ISSUE #22508: Error when try to generate a Bundle with a User
**Describe the bug**
When we go try to generate a bundle that contains any User we are getting this error

```
Caused by: java.lang.reflect.InaccessibleObjectException: Unable to make field private final jdk.internal.loader.BuiltinClassLoader jdk.internal.loader.BuiltinClassLoader.parent accessible: module java.base does not "opens jdk.internal.loader" to unnamed module @fadd4d07
	at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:340) ~[?:?]
	at java.lang.reflect.AccessibleObject.checkCanSetAccessible(AccessibleObject.java:280) ~[?:?]
	at java.lang.reflect.Field.checkCanSetAccessible(Field.java:176) ~[?:?]
	at java.lang.reflect.Field.setAccessible(Field.java:170) ~[?:?]
	at com.thoughtworks.xstream.converters.reflection.FieldDictionary.buildMap(FieldDictionary.java:148) ~[xstream-1.4.8.jar:1.4.8]
```

**To Reproduce**
1. Go to User portlet
2. Select the "Add to Bundle" option
3. Go to 'Publishing Queue' portlet and into the Bundle tab, select download Bundle

**Expected behavior**
Should generated the bundle without problem

### PR DESCRIPTION
**Pull request by freddyucv**
The bug was that it was trying to create the bundle.xml file and then it need to serialize the PushPublisherConfig class.

from the manifest file changes (long time ago) we really not need to create the bundle.xml anymore for no static bundle, so we really not need to serialize the  PushPublisherConfig class, it is supposed to create the manifest file instead.

but the User assets was not been adding into the manifest file that why the manifest file was not created

Also when just add User into the Bundle the Language Variable are not add into the bundle but in the Manifest file was reporting that they was added

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/publisher/pusher/PushPublisherConfig.java [hunk 1]
```diff
@@ -113,6 +113,21 @@ public void setSwitchIndexWhenDone(boolean switchIndexWhenDone) {
 		this.switchIndexWhenDone = switchIndexWhenDone;
 	}
 
+	public boolean justIncludesUsers() {
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/publisher/pusher/PushPublisherConfig.java:116]: javadoc

**Review by victoralfaro-dotcms — APPROVED** [review]
LGTM

File: dotCMS/src/main/java/com/dotcms/publisher/util/dependencies/DependencyManager.java [hunk 2]
```diff
@@ -44,6 +44,7 @@
 
 import org.apache.commons.lang.StringUtils;
 import org.apache.felix.framework.OSGIUtil;
+import org.jetbrains.annotations.NotNull;
```
Comment (dsilvam) [inline dotCMS/src/main/java/com/dotcms/publisher/util/dependencies/DependencyManager.java:47]: remove

Comment (freddyucv) [reply dotCMS/src/main/java/com/dotcms/publisher/util/dependencies/DependencyManager.java]: done https://github.com/dotCMS/core/pull/22511/commits/1aa7c71fe66cd096189c46b6f95ec7477a8cb60b

**Review by dsilvam — APPROVED** [review]

**Pull request merged**